server/src/main/java/org/spine3/server/event/EventBus.java

```diff
@@ -459,8 +460,10 @@ public Builder setEnricher(EventEnricher enricher) {
         }
 
         public EventBus build() {
-            checkState(storageFactory != null || eventStore != null,
-                       "Either storageFactory or eventStore must be set to build the EventBus instance");
+            final String message =
+                    "Either storageFactory or eventStore must be " +
+                    "set to build the EventBus instance";
```

Comment:
I'm using `+` sign after line since I've already received a comment opposite to yours. 
@alexander-yevsyukov can you please state a right way?